brainscore/benchmarks/rajalingham2018.py

```diff
@@ -23,15 +23,15 @@
             }"""
 
 
-class DicarloRajalingham2018I2n(BenchmarkBase):
-    def __init__(self):
-        self._metric = I2n()
+class _DicarloRajalingham2018(BenchmarkBase):
+    def __init__(self, metric, metric_name):
+        self._metric = metric
         self._fitting_stimuli = brainscore.get_stimulus_set('dicarlo.objectome.public')
         self._assembly = LazyLoad(lambda: load_assembly('private'))
         self._visual_degrees = 8
         self._number_of_trials = 2
-        super(DicarloRajalingham2018I2n, self).__init__(
-            identifier='dicarlo.Rajalingham2018-i2n', version=2,
+        super(_DicarloRajalingham2018, self).__init__(
+            identifier='dicarlo.Rajalingham2018-'+metric_name, version=2,
```

Comment:
should have spaces, i.e. `'dicarlo.Rajalingham2018-' + metric_name` -- you can also run pycharm's autoformat (ctrl+shift+f) for me